addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -111,48 +111,50 @@ def _get_partner_party_vals(self, partner, role):
         # With a Dutch supplier (NL), SchemeID may only contain 106 (Chamber of Commerce number) or 190 (OIN number).
         # [BR-NL-10] At a Dutch supplier, for a Dutch customer ( AccountingCustomerParty ) the customer registration
         # number must be filled with Chamber of Commerce or OIN. SchemeID may only contain 106 (Chamber of
         # Commerce number) or 190 (OIN number).
-        if partner.country_code == 'NL' and 'l10n_nl_oin' in partner._fields:
-            if partner.l10n_nl_oin:
+        if commercial_partner.country_code == 'NL' and 'l10n_nl_oin' in commercial_partner._fields:
+            if commercial_partner.l10n_nl_oin:
                 vals.update({
-                    'endpoint_id': partner.l10n_nl_oin,
+                    'endpoint_id': commercial_partner.l10n_nl_oin,
                     'endpoint_id_attrs': {'schemeID': '0190'},
                 })
-            elif partner.l10n_nl_kvk:
+            elif commercial_partner.l10n_nl_kvk:
                 vals.update({
-                    'endpoint_id': partner.l10n_nl_kvk,
+                    'endpoint_id': commercial_partner.l10n_nl_kvk,
                     'endpoint_id_attrs': {'schemeID': '0106'},
                 })
-        if partner.country_id.code == 'SG' and 'l10n_sg_unique_entity_number' in partner._fields:
+        if commercial_partner.country_id.code == 'SG' and 'l10n_sg_unique_entity_number' in commercial_partner._fields:
             vals.update({
-                'endpoint_id': partner.l10n_sg_unique_entity_number,
+                'endpoint_id': commercial_partner.l10n_sg_unique_entity_number,
                 'endpoint_id_attrs': {'schemeID': '0195'},
             })
-        if partner.country_id.code == "LU" and 'l10n_lu_peppol_identifier' in partner._fields and partner.l10n_lu_peppol_identifier:
-            vals['endpoint_id'] = partner.l10n_lu_peppol_identifier
+        if commercial_partner.country_id.code == "LU" and 'l10n_lu_peppol_identifier' in commercial_partner._fields and commercial_partner.l10n_lu_peppol_identifier:
+            vals['endpoint_id'] = commercial_partner.l10n_lu_peppol_identifier
         if partner.country_id.code == "SE" and partner.vat:
-            vals['endpoint_id'] = partner.vat.replace("SE", "")[:-2]
+            vals['endpoint_id'] = commercial_partner.vat.replace("SE", "")[:-2]
 
         return vals
 
     def _get_partner_party_identification_vals_list(self, partner):
         # EXTENDS account.edi.xml.ubl_21
         vals = super()._get_partner_party_identification_vals_list(partner)
 
-        if partner.country_code == 'NL' and 'l10n_nl_oin' in partner._fields:
-            endpoint = partner.l10n_nl_oin or partner.l10n_nl_kvk
+        commercial_partner = partner.commercial_partner_id
+        if commercial_partner.country_code == 'NL' and 'l10n_nl_oin' in commercial_partner._fields:
+            endpoint = commercial_partner.l10n_nl_oin or commercial_partner.l10n_nl_kvk
```

Comment:
If we're only using the commercial_partner_id in here, isn't easier to just call
`_get_partner_party_identification_vals_list(partner.commercial_partner_id)` ?